examples/with-graphql-tools/README.md

```diff
@@ -0,0 +1,11 @@
+## With graphql-tools example
+
+after Apollo team release new version of graphql-tools, we can merge our schema across between library so you can use both graphql-compose with graphql-tools
+
+## Run demo
+> Note: you need to build Graphql-compose before run this example
```

Comment:
Let remove this note and import `graphql-compose` explicitly from npm registry.

Reply:
I did try to install `graphql-compose` with NPM, but look like it does not allow to install package itself inside NPM project 🤔

maybe I will import `graphql-compose` but take a note for this issues ?

Reply:
Didn't know.

Ok, let import `graphql-compose` and add the note there.
Thanks.

Reply:
eh, my mistake It allow LOL 

maybe when I tried to install I ran it on graphql-compose root folder 😆

Reply:
well done, will write an article for this one

also I will keep adding more example to use this library, it will be useful for anyone who want to start to use this lib.

Reply:
Yep. Will be awesome.
Forming community around graphql-compose provides much more tools and stability.

I do not have enough time to do it myself.